packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/datacentre-usage-tile/DatacentreUsageTile.component.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  ActionMenu,
+  DashboardTile,
+  Description,
+} from '@ovh-ux/manager-react-components';
+import { ODS_ICON_NAME } from '@ovhcloud/ods-components';
+import IVcdDatacentre from '@/types/vcd-datacenter.interface';
+
+type TTileProps = {
+  vcdDatacentre: IVcdDatacentre;
+};
+
+export default function DatacentreUsageTile({
+  vcdDatacentre,
+}: Readonly<TTileProps>) {
+  const { t } = useTranslation('hpc-vmware-managed-vcd/datacentres');
```

Comment:
We could just put the translation keys at the root `useTranslation('datacentres')`